docs/src/main/sphinx/connector/ignite.rst

```diff
@@ -0,0 +1,156 @@
+====================
+Ignite connector
```

Comment:
Can you please mention what version of Ignite are supported?

Reply:
done